### ISSUE TITLE
Please bridge tg:@fedorazh <-> irc:#fedora-zh

### ISSUE THREAD

**Issue by alick** [RITlug/teleirc#52]
-    Organization / group name and website: FZUG, i.e. Fedora Chinese community. [website](https://zh.fedoracommunity.org/)
-    Telegram group URL: https://t.me/fedorazh
-    Your IRC channel: #feodra-zh

**Comment by mxa** [comment]
I don't think this is a bridging service. It's a software **you can run on your server** that is doing the bridging.

**Comment by jflory7** [comment]
@alick If you would like a new Telegram <=> IRC bridge for a Fedora community, please [file a Fedora Infrastructure ticket](https://pagure.io/fedora-infrastructure/) and tag me there. I can help you set one up inside of Fedora's infrastructure.
